coderabbit-demo/user-lookup.js

```
@@ -0,0 +1,26 @@
+'use strict'
+
+const sqlite3 = require('sqlite3')
+const crypto = require('crypto')
+
+const db = new sqlite3.Database('./app.db')
+
+const ADMIN_PASSWORD = 'S3cr3t-Admin-Pa55!'
```

ESLint

[error] 'ADMIN_PASSWORD' is assigned a value but never used.

(no-unused-vars)